tools/packaging/static-build/qemu/build-base-qemu.sh

```diff
@@ -31,7 +31,7 @@ info "Build ${qemu_repo} version: ${qemu_version}"
 
 http_proxy="${http_proxy:-}"
 https_proxy="${https_proxy:-}"
-prefix="${prefix:-"/opt/kata"}"
```

Comment:
So my understanding, which might well be wrong, is that for CC this script is called from build-static-qemu-cc.sh, which should already have the prefix exported?

https://github.com/kata-containers/kata-containers/blob/8330fdeadfd7fb92793a642a89e5de7fb2bbdafc/tools/packaging/static-build/qemu/build-static-qemu-cc.sh#L19

Reply:
Yep, that's exactly the case, @stevenhorsman!

Reply:
Please correct me if I am missing something @stevenhorsman @fidencio 
But based on my understanding since QEMU build is now moved to experimental, we will now use `build-static-qemu-experimental.sh` instead of `build-static-qemu-cc.sh`
This is being called from here: https://github.com/kata-containers/kata-containers/blob/d742fe37a92a7be600f2fb2d676108697b91abad/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh#L810